ballerina/listener.bal

```diff
@@ -41,6 +45,16 @@ public class Listener {
             self.httpListener = listenTo;
         }
         self.wsListener = ();
+        self.graphiql = {};
+        string host = configuration.host;
+        int port = self.httpListener.getPort();
+        if configuration.secureSocket is () {
+            self.httpEndpoint = string `http://${host}:${port}`;
+            self.websocketEndpoint = string `ws://${host}:${port}`;
+        } else {
+            self.httpEndpoint = string `https://${host}:${port}`;
+            self.websocketEndpoint = string `wss://${host}:${port}`;
```

Comment:
Do we have a test for the secured WebSocket endpoint?

Reply:
Currently we don't have a test